src/lib/Microsoft.Fx.Portability.Reports.DGML/DGMLManager.cs

```diff
@@ -120,7 +120,7 @@ internal void Save(Stream stream)
             {
                 file.Save(ms);
                 ms.Position = 0;
-                ms.CopyTo(stream);
+                ms.CopyToAsync(stream);
```

Comment:
The entire chain will need to be async. Can't just change it here

Reply:
Fixed